index.html

```diff
@@ -0,0 +1,233 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>Quiz</title>
+  <link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+  <article id="Widgets">
+    <div id="cards">
+
+      <div class="card" id="widget-A">
+        <div class="card-header-block">
+          <header class="card-header">
```

Comment:
Thanks.  I'll get those done shortly.


On Wed, Apr 20, 2016 at 3:55 PM, Greg Korte <notifications@github.com>
wrote:

> In index.html
> <https://github.com/nss-day-cohort-13/static-web-html-css-quiz-jbrooks036/pull/1#discussion_r60485712>
> :
>
> > @@ -0,0 +1,233 @@
> > +<!DOCTYPE html>
> > +<html>
> > +<head>
> > +  <meta charset="utf-8">
> > +  <meta http-equiv="X-UA-Compatible" content="IE=edge">
> > +  <title>Quiz</title>
> > +  <link rel="stylesheet" href="quiz.css">
> > +</head>
> > +<body>
> > +  <article id="Widgets">
> > +    <div id="cards">
> > +
> > +      <div class="card" id="widget-A">
> > +        <div class="card-header-block">
> > +          <header class="card-header">
>
> Each <header> element should hold an <h1> element with the title in it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/nss-day-cohort-13/static-web-html-css-quiz-jbrooks036/pull/1/files/7fe245c6b8a9a27f07eb83871a8207dc8d442615#r60485712>
>